CMakeLists.txt

```diff
@@ -310,12 +315,13 @@ endif()
 #if(APPLE)
```

Comment:
I think the macos case can go away: As far as I can see, it's a copy+paste artifact from sdl12-compat

Reply:
Done.
Looking in the src/main folder, only gdk, haiku, uikit and winrt have shared SDL2 libraries and can thus have sdl2-compat support.